test/langtools/jdk/javadoc/tool/CheckManPageOptions.java

```diff
@@ -66,7 +66,7 @@ public static void main(String... args) throws Exception {
 
     static final PrintStream out = System.err;
 
-    List<String> MISSING_IN_MAN_PAGE = List.of("--spec-base-url");
+    List<String> MISSING_IN_MAN_PAGE = List.of();
```

Comment:
What's that about? It seems unrelated.

Reply:
It's as if this mainline commit somehow got into this PR: https://github.com/openjdk/jdk/commit/82f9819eaccd091c9d3a0b89979ddc13b1ef761c

Reply:
I'll investigate, but it looks like just a side effect of importing the latest troff man page.

Reply:
It's now been merged in.